### PR TITLE
Added our-program-section- HTML

### PR DESCRIPTION
**Pull request by JTLiberona**
Elements added:

- H2.
- Paragraph.
- Button.
- Image.

### CODE REVIEW

**Review by katsmamina — COMMENTED** [review]
I don't know why the first part looks deleted and then inserted here :)

**Comment by JTLiberona** [comment]
Do you still have the same problem? I saw the file and there is everything it was but in new files now.

**Comment by katsmamina** [comment]
I think I need to watch again the today's lessons before accepting :)

File: index.html [hunk 1]
```diff
@@ -1,62 +1,90 @@
...
+    </header>
+    <!--Main-->
+    <main>
+      <!--Our program-->
+      <section class="our-program">
+        <div class="op-container-2">
+          <div class="op-container-3">
+            <!-- Left Container -->
+            <div class="our-program-left">
+              <h3>Our program</h3>
```
Comment (katsmamina) [inline index.html:65]: I marked the text of the same size as h2. I suppose there is only h1 above, which is the "Helping new talent ..." text. h3 is for example "community' in OUR CORE VALUES section.

File: index.html [hunk 2]
```diff
@@ -1,62 +1,90 @@
...
+              <p>
+                Our <b>8-month</b> program is divided into <b>12 modules</b>.
+                The last module is a
+                <b>6-weeks intensive Final Project</b> where students work on a
+                real project with a real partner. Our training takes place at
+                <b>BeCentral</b> (Brussels) every <b>Sunday</b>. During the
+                week, students are supported online in their <b>self-study</b>,
+                homework and peer learning.
+              </p>
+              <button type="button">Discover more</button>
```
Comment (katsmamina) [inline index.html:75]: That's a good point. I made my button as a link, so I will remake one.

Comment (rayanejsilva) [reply index.html:75]: @katsmamina it is not a problem to make a button with a tag, you can style it to look like a button. I think Jose forgot to add the link to the program page here.

File: index.html (hunk 1 above)
Comment (JTLiberona) [reply index.html:65]: Yes. It is an h2. I made changes for homepage but not for this one. I will do it.

File: index.html (hunk 2 above)
Comment (JTLiberona) [reply index.html:75]: Yes. There is no link. I will modify it.
And regardig the button. As Rayana saids it is an option to anchor a button into a link. I saw that in Our program section of the HYF site when i codereversed it.

File: index.html [hunk 3]
```diff
@@ -59,14 +59,14 @@
...
-      <section class="our-program">
+     <!--Our program-->
+      <section class="op-container-1">
         <div class="op-container-2">
           <div class="op-container-3">
             <!-- Left Container -->
             <div class="our-program-left">
-              <h3>Our program</h3>
-              <p>
+              <h2 class="op-title">Our program</h2>
```
Comment (JTLiberona) [inline index.html:68]: Change of class h3 > h2 done.

File: index.html [hunk 4]
```diff
@@ -75,10 +75,26 @@ <h3>Our program</h3>
                 week, students are supported online in their <b>self-study</b>,
                 homework and peer learning.
               </p>
-              <button type="button">Discover more</button>
+              <a href="program.html">
```
Comment (JTLiberona) [inline index.html:78]: Link/Button has the link to the program's site.

**Comment by JTLiberona** [comment]
The requested changes have been updated.
- h3 > h2 
- The link redirect to program.html

Please take a look.
:)

**Review by katsmamina — APPROVED** [review]

**Pull request merged**